EcpBtBot/DialogFlow/API.py

```diff
@@ -13,60 +15,22 @@
         )
     )
 
-CLIENT_ACCESS_TOKEN_SePresenter = '868aec3d1e0f408392c3c2993fb05cfa'
 
 
-class API_SePresenter:
-    def __init__(self):
+class API_Connection:
+    def __init__(self,subject):
+        """
         #Connexion api
-        self.ai=apiai.ApiAI(CLIENT_ACCESS_TOKEN_SePresenter)
+        """
+        CLIENT_ACCESS_TOKEN = tokens[subject]
```

Comment:
again, you seem to have a global variable here... better to also pass `tokens` as argument of the class.

Reply:
ok